mlflow/store/file_store.py

```diff
@@ -31,6 +31,12 @@ def _default_root_dir():
     return get_env(_TRACKING_DIR_ENV_VAR) or os.path.abspath(DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH)
 
 
+def _read_persisted_experiment_dict(experiment_dict):
```

Comment:
Let's add a comment on why we have to read this specially (i.e., to read legacy experiments when it was serialized as in an int)